### PR TITLE
vPC documentation in README.md

### PR DESCRIPTION
**Pull request by dcheriancisco**
The docs update cisco_interface with 2 new params: vpc_id and vpc_peer_link and adds a new type: cisco_vpc_domain

### CODE REVIEW

File: lib/puppet/provider/cisco_ace/nxapi.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 #
 # The NXAPI provider for cisco_ace.
 #
-# July, 2015
+# January 2016
```
Comment (mikewiebe) [inline lib/puppet/provider/cisco_ace/nxapi.rb]: Why do you have `cisco_ace` diffs here?  Was this merge noise?

Comment (dcheriancisco) [inline lib/puppet/provider/cisco_ace/nxapi.rb]: No, I think this is noise

Comment (robert-w-gries) [inline lib/puppet/provider/cisco_ace/nxapi.rb]: We'll need the noise cleaned up before we can merge this PR.  Please remove the following commits from your PR:

* bdf0e0e3e20e12195663edfcadd29b8c9dc5d027
* 89621e2e3e97bc673105fd6e2a447d5cdfc59272

I recommend you checkout `release_1.2.0` branch on your fork, pull in only your doc change commits, then submit a new PR.

**Comment by dcheriancisco** [comment]
please use #151

**Pull request closed without merging**